deploy/sre-prometheus/100-telemeter-client.PrometheusRule.yaml

```diff
@@ -11,8 +11,10 @@ spec:
   - name: sre-telemeter-client
     rules:
     - alert: MetricsClientSendFailingSRE
```

Comment:
```suggestion
    - alert: MetricsClientSendFailingSRE
    # metricsclient sends 1 request/5m, 12/60m, 288/day, 8064/28days.
    # SLO=90%, Error Budget=10% or 804 failed requests.
    # Send an alert when 2% of Error budget (16.2 requests) is burned within a 2 hour window. Why 2 hours? Because the combination of request frequency and SLO @90% requires atleast 2 hours to breach a 2% burn. 
    # The short window of 10 minutes allows for quick recovery once the issue is resolved. 
```

Reply:
cc @cblecker

Reply:
lgtm